extensions/positron-python/package.json

```
@@ -529,6 +529,24 @@
                     "scope": "machine",
                     "type": "string"
                 },
+                "python.environmentProviders.enable": {
+                    "default": {
+                        "Venv": true,
+                        "Conda": true
+                    },
+                    "examples": [
+                        {
+                            "Venv": true,
+                            "Conda": true
+                        }
+                    ],
+                    "markdownDescription": "%python.environmentProviders.enable.markdownDescription%",
+                    "scope": "machine",
+                    "type": "object",
+                    "additionalProperties": {
```

```suggestion
                    "additionalProperties": false
```

(also needed if you're changing settings to match the screenshot)